storage/filesystem/internal/dotgit/dotgit.go

```diff
@@ -103,3 +106,70 @@ func (d *DotGit) Idxfile() (fs.FS, string, error) {
 
 	return nil, "", ErrIdxNotFound
 }
+
+// Objectfiles returns the hash slice of the objects files in the "objects/"
+func (d *DotGit) Objectfiles() (fs.FS, []core.Hash, error) {
+	objsDir := d.fs.Join(d.path, "objects")
+
+	files, err := d.fs.ReadDir(objsDir)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	var objects []core.Hash
+	for _, f := range files {
+		if f.IsDir() && len(f.Name()) == 2 && isHex(f.Name()) {
+			objDir := f.Name()
```

Comment:
The names of the vars `objDir` and `objsDir` are too similar and it is confusing.

Can you rename `objDir` as `d`? (that way it would be coherent with the `f` var naming and quite diferent form `objsDir`. You can rename the receiver as `dg`, for example.